app/views/support_interface/unauthorized.html.erb

```diff
@@ -0,0 +1,19 @@
+<% content_for :title, t('page_titles.application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Your account is not authorized
+    </h1>
+    <p class="govuk-body">
+      You’re seeing this page because you’ve signed in
+      to the <%= t('service_name.apply') %> service but your account
+      is not authorized to view the support interface.
```

Comment:
Ought we to be a bit more uninformative about what's here?

Reply:
We can say 'not authorized to view this page'.?